rules/retrieve.smk

```diff
@@ -1015,59 +1015,145 @@ if (LAU_REGIONS_DATASET := dataset_version("lau_regions"))["source"] in [
         run:
             copy2(input["lau_regions"], output["zip"])
 
+
+if (SEAWATER_TEMPERATURE_DATASET := dataset_version("seawater_temperature"))[
+    "source"
+] in [
+    "archive",
+]:
+
     rule retrieve_seawater_temperature:
```

Comment:
Wrongly named? I think this is supposed to be the pendant to the `build` rule for `SEAWATER_TEMPERATURE_COPERNICUSMARINE_DATASET`?